src/blockchain/web3_extentions/lido_contracts.py

```diff
@@ -54,6 +55,15 @@ def _load_contracts(self):
         self._load_staking_router()
         self._load_dsm()
 
+        if variables.is_mellow_strategy_set():
```

Comment:
Let's move it to separate function like above

Reply:
Done